Pipelines/Templates/deploy-Solution.yml

```diff
@@ -129,9 +129,17 @@ steps:
   displayName: 'Get managed solution zip path'
   condition: and(succeeded(), ne('${{parameters.importUnmanaged}}', 'true'))
 
-- pwsh: |    
-    . "$env:POWERSHELLPATH/set-trigger-solution-upgrade.ps1"
-    Set-TriggerSolutionUpgrade '$(TriggerSolutionUpgrade)' '$(System.TeamFoundationCollectionUri)' '$(System.TeamProjectId)' '$(Build.Repository.Name)' '$(Build.Repository.Provider)' '$(Build.SourceVersion)' '$(Build.Reason)' '$(GitHubRepo)' '$(GitHubPAT)'
+- pwsh: |   
+    # Read the Trigger Solution Upgrade flag from pipeline variable
+    Write-Host "$TriggerSolutionUpgrade - $(TriggerSolutionUpgrade)"
+    Write-Output "##vso[task.setvariable variable=TriggerSolutionUpgrade;isOutput=true]$(TriggerSolutionUpgrade)"
+
+    ## If Solution Upgrade was not set at Canvas App, check whether PR has Tag
```

Comment:
I thought we agreed to keep the PR Label functionality for the time being as a fallback in any case where TriggerSolutionUpgrade isn't set. I'm okay with removing it all together, but if we do we'll need to make sure we update the docs to reflect this change and list this as a potentially breaking change in the release. Would also like to just remove any code associated with the PR Tag based approach rather than commenting it out.

Reply:
Enabled fallback option (i.e., If Solution Upgrade was not set at Canvas App Deployment Settings screen, check whether PR has Tag)